README.md

```diff
@@ -84,14 +85,14 @@ To register the application with the Connector service, you need to create and s
 
 ### 6LoWPAN ND and Thread settings
 
-First, you need to select the RF driver to be used by the 6LoWPAN/Thread stack. This example supports [AT86RF233/212B](https://github.com/ARMmbed/atmel-rf-driver) and [NXP-MCR20a](https://github.com/ARMmbed/mcr20a-rf-driver) radio shields.
+First, you need to select the RF driver to be used by the 6LoWPAN/Thread stack. This example supports [AT86RF233/212B](https://github.com/ARMmbed/atmel-rf-driver), [NXP-MCR20a](https://github.com/ARMmbed/mcr20a-rf-driver), and [X-NUCLEO-IDS01A4](https://github.com/ARMmbed/stm-spirit1-rf-driver) (*a.k.a.* Spirit1) radio shields.
 
-To add the Atmel driver to you application from command line, call: `mbed add https://github.com/ARMmbed/atmel-rf-driver`.
+To add the Atmel driver to your application from command line, call: `mbed add https://github.com/ARMmbed/atmel-rf-driver`.
```

Comment:
This part is now a bit outdated - the new RFs should be added to the easy-connect repo, not directly to the app itself anymore.